### PR TITLE
Back up & restore old git hooks

### PR DESCRIPTION
**Pull request by SerkanYildiz**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | kind of
| BC breaks?    | no
| Deprecations? | no
| Documented?   | n/a
| Fixed tickets | #416

### CODE REVIEW

File: src/Console/Command/Git/InitCommand.php [hunk 1]
```diff
@@ -176,4 +192,38 @@ protected function paths()
...
+
+    /**
+     * @param string          $hookPath
+     * @param SplFileObject   $gitHookFile
+     *
+     * @param OutputInterface $output
+     *
+     * @return void
+     *
+     * @throws \Symfony\Component\Filesystem\Exception\IOException
```
Comment (Landerstraeten) [inline src/Console/Command/Git/InitCommand.php]: FQDN

File: src/Console/Command/Git/InitCommand.php [hunk 2]
```diff
@@ -75,38 +76,53 @@ protected function configure()
      * @param OutputInterface $output
      *
      * @return int|void
+     *
+     * @throws \Symfony\Component\Filesystem\Exception\IOException
```
Comment (Landerstraeten) [inline src/Console/Command/Git/InitCommand.php]: FQDN

**Comment by sanmai** [comment]
This is good, but it does not fix #416. With this PR or without grumphp will still corrupt my hooks.

**Comment by Landerstraeten** [comment]
@SerkanYildiz. We will close this PR. There is a note in the README that your existing git hooks will be overwritten. Thanks for the effort!

**Pull request closed without merging**